project-based/README.md

```diff
@@ -0,0 +1,25 @@
+# Dockerfile for Zen Theme.
+
+### Usage (Without Makefile)
+
+```
+docker run --rm -t -u 1000:1000 -v $(pwd):/work abc/xyz 
+```
+
+### Steps to build a new image
+
+_Note: this is just a quick guide to get started using this new image_
+
+- Login to gitlab so we can use gitlab registry `make login` or `docker login ...`
+- Use the same `package.json` as the one located inside the project repo 
+- You can set a project name in the `Makefile`
+- Build the container with `make build` 
+- Push it to gitlab with `make push`
+- Update the `.env` file from your project to use this new image
+
+### Why this image instead of the other ones?
+
+- Based on node.js 9.x
+- Easier to mantain: doesn't require packages to be hardcoded in the Dockerfile, we only mantain one package.json
```

Comment:
But it still require to rebuild image when packages changed

Reply:
That's right but I still think its better than we used to have (we can have easy steps to update images using the same package.json)